assets/react-hooks/index.test.js

```diff
@@ -0,0 +1,77 @@
+import { render, fireEvent } from '@testing-library/react';
```

Comment:
I like to use the tests together with the files because makes the maintenance easier. But if someone has a different opinion, we can change and make a mirror of the assets to create the tests.

Reply:
Yup good for me.